### PR TITLE
[TRAH] Sergei / TRAH 3146 / Add performance metrics for trader's hub new repo

### PR DESCRIPTION
**Pull request by sergei-deriv**
- add `@deriv-com/analytics` package
- add `startInitTimers` to main and create performance metrics file
- add all possible metrics for now (except `cTrader` and `DerivX` creation, and `cashier` load)
- change `mutate` to `mutateAsync` in `useMT5AccountHandler` hook because we need to know when the request completed

### CODE REVIEW

File: src/components/CurrencySwitcher/CurrencySwitcher.tsx [hunk 1]
```diff
@@ -36,7 +38,10 @@ const AccountActionButton = ({ balance, isDemo }: AccountActionButtonProps) => {
                         loginid: activeLoginid,
                     });
                 } else {
-                    navigate('/cashier/deposit');
+                    if (currencyConfig?.currency?.isCrypto)
+                        startPerformanceEventTimer('load_crypto_deposit_cashier_time');
+                    else startPerformanceEventTimer('load_fiat_deposit_cashier_time');
+                    navigate('/cashier/deposit#deposit');
```
Comment (habib-deriv) [inline src/components/CurrencySwitcher/CurrencySwitcher.tsx:44]: As traders hub is in new repo, our url is changed too. We need to use this url
https://app.deriv.com/cashier/deposit#deposit

File: src/components/TotalAssets/TotalAssets.tsx [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+import { setPerformanceValue } from '@/utils';
+
+import { TotalAssetsLoader } from '../Loaders';
 
 const TotalAssets = () => {
     const { data: activeDerivTradingAccount } = useActiveDerivTradingAccount();
     const { formattedTotalBalance } = useTotalAssets();
+    const { isMobile } = useDevice();
+
+    // need to add more conditions to show the loader and wait until all accounts are measured
```
Comment (habib-deriv) [inline src/components/TotalAssets/TotalAssets.tsx:15]: Lets add TODO in the comment

File: src/assets/svgs/personal-details-example.svg
Comment (habib-deriv) [inline src/assets/svgs/personal-details-example.svg:1]: Can we use an optimized version of this svg?
Maybe can convert it it jpg and optimize using tinypng

Comment (thisyahlen-deriv) [reply src/assets/svgs/personal-details-example.svg:1]: I already did it @sergei-deriv. Used jpg instead

Comment (sergei-deriv) [reply src/assets/svgs/personal-details-example.svg:1]: @thisyahlen-deriv Do you mean we can delete this image?

Comment (thisyahlen-deriv) [reply src/assets/svgs/personal-details-example.svg:1]: yes @sergei-deriv i already deleted it. Its no mo

**Review by thisyahlen-deriv — APPROVED** [review]

**Pull request merged**